Sources/Common/Type/SDKWrapperConfig.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+
+/// Metadata about a SDK wrapper/bridge made to use this Customer.io SDK.
+public struct SdkWrapperConfig {
```

Comment:
I like the idea of adding another property to this struct:

```swift
public struct SdkWrapperConfig {
  let bundleId: String
}
```

Then, for developers who wrap our SDK, they provide a bundle ID such as "io.customer.reactnative", "com.github.cool_customerio_rn_sdk". We can use this value mostly in log statements. Can be useful during customer debugging in case they are using a 3rd party/unofficial SDK made by someone else.

Reply:
Since we do not encourage customers to create their own wrappers around our SDK, this feature is not needed.